lib/hanami/cli/commands/app/generate/action.rb

```diff
@@ -106,7 +112,8 @@ def call(
                 raise InvalidActionNameError.new(name)
               end
 
-              generator.call(app.namespace, controller, action, url, http, format, skip_view, skip_route, slice, context: context)
+              generator.call(app.namespace, controller, action, url, http, format, skip_view, skip_route, template, slice,
```

Comment:
It feels weird with so many positional arguments. Should I convert this to keyword?